index.html

```diff
@@ -29,8 +29,7 @@
     <!-- Theme CSS -->
     <link href="css/agency.css" rel="stylesheet">
     <link rel="stylesheet" href="css/community.css">
-
-
+    <link rel="stylesheet" href="./css/cleverhare.css">
```

Comment:
Please rename the CSS file to something more relevant, that might not result in any future fellow contributors
For Example : 
````suggestion
    <link rel="stylesheet" href="./css/Issues_hover.css">
````

Reply:
Sure